cypress/integration/seededFlows/adminFlows/badges/badgeAchievements.spec.js

```diff
@@ -5,14 +5,83 @@ describe('Badge Achievements', () => {
 
     cy.get('@user').then((user) => {
       cy.loginAndVisit(user, '/admin/content_manager/badge_achievements');
+
+      cy.findByRole('table').within(() => {
+        cy.findByRole('button', { name: 'Remove' }).click();
+      });
     });
   });
 
-  it('delete a badge achievement', () => {
-    cy.findByText('Remove').as('removeBadgeButton');
+  describe('delete a badge achievement', () => {
+    it('should display confirmation modal', () => {
+      cy.get('.crayons-modal__box > header > p')
```

Comment:
If we refactor the modal to the Preact component, we'll be able to target this content a bit more easily, without relying on the class names 🙂